### PR TITLE
fix(ssl): fix API returns the certificate file name instead of the certificate content

### PR DESCRIPTION
**Pull request by tigercl**

### CODE REVIEW

**Comment by zmstone** [comment]
Test cases are failing.

File: apps/emqx_authn/src/emqx_authn_api.erl [hunk 1]
```diff
@@ -781,16 +781,16 @@ fill_defaults(Configs) when is_list(Configs) ->
 fill_defaults(Config) ->
     emqx_authn:check_config(Config, #{only_fill_defaults => true}).
 
-convert_certs(#{<<"ssl">> := SSLOpts} = Config) ->
+convert_certs(#{ssl := SSLOpts} = Config) ->
```
Comment (zmstone) [inline apps/emqx_authn/src/emqx_authn_api.erl:784]: doesn't this function work on the raw config?

File: apps/emqx_authn/src/emqx_authn_api.erl [hunk 2]
```diff
@@ -781,16 +781,16 @@ fill_defaults(Configs) when is_list(Configs) ->
...
     NSSLOpts = lists:foldl(fun(K, Acc) ->
                                case maps:get(K, Acc, undefined) of
                                    undefined -> Acc;
                                    Filename ->
                                        {ok, Bin} = file:read_file(Filename),
                                        Acc#{K => Bin}
```
Comment (zmstone) [inline apps/emqx_authn/src/emqx_authn_api.erl:790]: If the intension is to *NOT* return file content in the HTTP response, shouldn't we just skip doing this?

**Review by zmstone — CHANGES_REQUESTED** [review]

File: apps/emqx_authn/src/emqx_authn_api.erl (hunk 1 above)
Comment (zmstone) [reply apps/emqx_authn/src/emqx_authn_api.erl:848]: for the record: it is indeed raw config as input, but filled with defaults, and checked with `atom_keys` option

**Review by terry-xiaoyu — APPROVED** [review]

**Pull request merged**